src/mpid/ch4/netmod/ofi/ofi_events.c

```diff
@@ -100,7 +100,7 @@ static int peek_event(struct fi_cq_tagged_entry *wc, MPIR_Request * rreq)
     /* util_id should be the last thing to change in rreq. Reason is
      * we use util_id to indicate peek_event has completed and all the
      * relevant values have been copied to rreq. */
-    MPIDI_OFI_REQUEST(rreq, util_id) = MPIDI_OFI_PEEK_FOUND;
+    MPL_atomic_release_store_int(&(MPIDI_OFI_REQUEST(rreq, util_id)), MPIDI_OFI_PEEK_FOUND);
```

Comment:
You can just use `MPL_atomic_store_int`. We added convenience wrappers for the common case (acquire/release).

Reply:
Okay, I'll change it to `MPL_atomic_store_int`.